lib/screens/login_screen.dart

```diff
@@ -95,60 +88,90 @@ class _LoginScreenState extends State<LoginScreen>
     return Scaffold(
       appBar: AppBar(),
       body: Center(
-        child: Column(
-          children: <Widget>[
-            ElevatedButton(
-              onPressed: () {
-                _controller.forward();
-                // Navigator.of(context).pushNamed('/mainscreen');
-              },
-              child: const Text('log in'),
-            ),
-            SlideTransition(
-              position: _offsetAnimation,
-              child: FadeTransition(
-                opacity: _controller,
-                child: Column(
-                  mainAxisSize: MainAxisSize.max,
-                  children: <Widget>[
-                    SizedBox(
-                      width: 200,
-                      child: TextField(
-                        controller: _loginController,
-                        decoration: const InputDecoration(
-                          border: OutlineInputBorder(),
-                          labelText: 'Login',
-                        ),
-                      ),
+        child: SingleChildScrollView(
```

Comment:
I would suggest, you use PicosBody and PicosScreenFrame, if nothing speaks against it.